projects/kit/components/input-phone/input-phone.component.ts

```diff
@@ -239,7 +239,6 @@ export class TuiInputPhoneComponent
 
     override writeValue(value: string | null): void {
         super.writeValue(value);
-        this.updateSearch('');
```

Comment:
I think it's probably needed. The issue is calling `writeValue` unnecessarily in isoCode setter here:
https://github.com/Tinkoff/taiga-ui/blob/main/projects/kit/components/input-phone-international/input-phone-international.component.ts#L78
You need an early return to skip the setter if `this.countryIsoCode === code`

Reply:
I'll research that. thanks

Reply:
Yep, you're right.